setup.py

```diff
@@ -143,7 +141,7 @@ def run_tests(self):
 
 setup(
     name='openhtf',
-    version='1.0',
+    version='1.1',
```

Comment:
we haven't been updating this because it's not on PyPI yet. I don't think we want to update it until we've got releases happening (tags, a PyPI upload process, etc)